packages/atlas/src/components/_inputs/RadioButton/RadioButton.tsx

```diff
@@ -3,25 +3,19 @@ import React from 'react'
 import { RadioAndCheckboxBase } from '../RadioAndCheckboxBase'
 import { RadioInput } from '../RadioInput'
 
-type RadioButtonProps = Partial<{
+export type RadioButtonProps = Partial<{
   selectedValue: string | number
   label: string
-  helperText?: string
+  caption?: string
   className?: string
   error?: boolean
 }> &
   React.InputHTMLAttributes<HTMLInputElement>
 
 export const RadioButton = React.forwardRef<HTMLInputElement, RadioButtonProps>(
-  ({ value, selectedValue, label, helperText, className, disabled, error, onChange, ...props }, ref) => {
+  ({ value, selectedValue, label, caption, className, disabled, error, onChange, ...props }, ref) => {
     return (
-      <RadioAndCheckboxBase
-        disabled={disabled}
-        className={className}
-        label={label}
-        error={error}
-        helperText={helperText}
-      >
+      <RadioAndCheckboxBase disabled={disabled} className={className} label={label} error={error} helperText={caption}>
```

Comment:
Let's also rename helperText to caption here. Is there a reason why we keep that unchanged?